src/ui/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -17,20 +17,22 @@ VPNFlickable {
 
     VPNMenu {
         id: menu
-        title: VPNl18n.tr(VPNl18n.SettingsDevTitle)
+        // Do not translate this string!
+        title: "Developer Options"
         isSettingsView: false
     }
 
     VPNCheckBoxRow {
         id: developerUnlock
-        objectName: "settingsDeveloperUnlock"
 
         anchors.top: menu.bottom
         anchors.topMargin: Theme.windowMargin
         width: parent.width - Theme.windowMargin
 
-        labelText:  VPNl18n.tr(VPNl18n.SettingsDevShowOptionTitle)
-        subLabelText: VPNl18n.tr(VPNl18n.SettingsDevShowOptionSubtitle)
+        // Do not translate this string!
+        labelText:  "Show Developer Options"
```

Comment:
I would keep all the strings in this file translatable (also considering they're already translated).

I don't expect them to change, they're clear, and it might be useful for users who accidentally enable this menu (not that it's easy).